problems/4.arrays.js

```diff
@@ -12,8 +16,8 @@ function firstAndLastAverage() {
 * @param {number[]} arr - The input array
 * @returns {number[]} - The duplicated array
 */
-function duplicateArray() {
-
+function duplicateArray(arr) {
+return(arr.concat (arr))
```

Comment:
Make sure to use a code formatter when you code like prettier.  Looks good besides that.